language-design.md

```diff
@@ -116,6 +314,11 @@ Briefly describe how you split up the work for this assignment.
 
 **Response**
 
-
+First, we did the readings separately. Then, we got together and talked about
+each of the questions in turn. We came up with parts of the texts we wanted to
+cite, and drafted a few points for each question together. After that, we
+agreed on a split for writing up questions and wrote up 3.5 answers each
+individually. Finally, we got back together, talked again about a few issues,
+and now we're done!
 
```

Comment:
haha Robin, thanks!

I liked that your critique raised deeper questions about the response and brought in outside examples (ex the Twitter backwards compatibility thing).